icu4c/source/common/brkiter.cpp

```diff
@@ -439,8 +439,8 @@ BreakIterator::makeInstance(const Locale& loc, int32_t kind, UErrorCode& status)
                 uprv_strcat(lb_lw, "_");
                 uprv_strcat(lb_lw, value.data());
             }
-            // lw=phrase is only supported in Japanese.
-            if (uprv_strcmp(loc.getLanguage(), "ja") == 0) {
+            // lw=phrase is only supported in Japanese and Korean
+            if (uprv_strcmp(loc.getLanguage(), "ja") == 0 || uprv_strcmp(loc.getLanguage(), "ko") == 0) {
```

Comment:
This needs to be done for ICU4J as well.

The new data will end up on the J side  at some point anyway, independently of this PR,  the first time anyone refreshes J data from ICU4C. Best to keep everything in sync, and do Java at the same time as C.

Reply:
Done in the latest PR.